src/include/zf/zf_stack.h

```diff
@@ -167,5 +167,81 @@ enum zf_stack_feature {
 */
 ZF_LIBENTRY int zf_stack_query_feature(struct zf_stack* stack, enum zf_stack_feature feature);
 
+/*! \brief Layout for a field of statistics */
```

Comment:
not sure zf_stack.h is correct place for all this.  I would consider new zf_stats.h (and add to zf.h)